### DISCUSSION TITLE
Failed to start watcher

### DISCUSSION THREAD

**Discussion by R3dFoxxx** [thalesgroup-cert/Watcher#108] [Q&A] [answered]
When trying to start Watcher using the latest version as recommended in the file, the system does not start and displays an error.
![Watcher](https://github.com/thalesgroup-cert/Watcher/assets/124746319/697ccf31-93e2-4d35-b206-bef1cc69a32b)

**Comment by shayol33** [comment]
i'm having same issue here
have you been able to fix it

**Comment by Felix83000** [comment, marked as answer]
Hi there,

This issue happens because your are using the latest version of mysql.

We have released a new version of Watcher here: https://github.com/thalesgroup-cert/Watcher/releases/tag/v2.0

Please install it with the newest docker-compose.yml 

Kind regards
